### PR TITLE
Added slang

### PR DESCRIPTION
**Pull request by danitacodes**
Closes #

### What new changes did you make? Tick all applicable boxes
- [x ] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Describe the new changes you added.
I added the following slang:
ifyp - I Feel Your Pain
imu - I Miss U
wywh - Wish You Were Here
ssdd - Same Stuff, Different Day
omdb - Over My Dead Body
soml - Story of My Life

### CODE REVIEW

File: public/server/db.json [hunk 1]
```diff
@@ -134,6 +134,34 @@
   "pov": {
     "definition": "Point Of View"
   },
+
```
Comment (Njong392) [inline public/server/db.json:137]: You didn't need to add these to the `db.json` folder any longer. I think I'll get rid of the folder soon anyways.

File: public/server/db/ifyp.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Njong392) [inline public/server/db/ifyp.json:1]: These new abbreviations look good! Thank you so much 🎉

**Comment by Njong392** [comment]
Whhoops. Seems like this pull request brought along some merge conflicts in the `CHANGELOG.md`. To fix this, I'd suggest;

- Syncing your fork to this upstream repo
- Running a git `git pull` to then sync your local repo with your fork
- Make necessary changes I suggested, that is, removing the additions you made to `db.json` folder
- If you can, when committing new changes, don't commit `package.json` as it shows here that you made changes to it too, even though you manually didn't.
- You can then push your changes and update this PR.

If you need help with anything,please feel free to ask.

File: public/server/db.json (hunk 1 above)
Comment (danitacodes) [reply public/server/db.json:137]: Hi, I didn't add these BUT it wouldn't let me continue without fixing the merge conflicts. So, I just took the changes.

**Comment by danitacodes** [comment]
Hi, I want to sync but it is saying there are 5 commits ahead and I would have to discard them. Should I?

**Comment by Njong392** [comment]
This all looks new to me, and I have tried googling but nothing useful comes up. Let's try together haha. Discard the changes, sync fork, and check if the changes you made (the abbreviations you added), are still present. If they aren't, then you can add them and update the PR.

If anything goes bad, we'll see what to do. Apologies for being clueless rn lol.

**Pull request closed without merging**